### PR TITLE
Fetch latest block transactions using getrawmempool for bitcoin-like blockchains (closes #1621)

### LINKED ISSUE #1621: Restore walletnotify event in Peatio 1.9 for bitcoin-like blockchains
The new blockchain daemon intruduced in Peatio 1.9 is adding to every deposit an delay of 15-30 minutes depending on the blockchain type.

For example in Bitcoin TestNET the deposit is appearing 15 minutes after the transaction is received from the LOCAL HOT wallet.

With the `walletnotify` event into the `bitcoin.conf` file configured the notification to the Peatio UI for an incoming transaction was immediate.

Can the `walletnotify` evend added again in Peatio 1.9??
Is there anybody working on adding again the `peatio.deposit.coin` message queue to be used with the `walletnotify` event??

```walletnotify=/usr/local/sbin/rabbitmqadmin publish routing_key=peatio.deposit.coin payload='{"txid":"%s", "currency":"btc"}'
```

### PR DESCRIPTION
**Pull request by dinesh-skyach**
Added wallet notify for bitcoin like currencies

### CODE REVIEW

File: app/services/blockchain_service/bitcoin.rb [hunk 1]
```diff
@@ -84,6 +85,14 @@ def build_withdrawals(block_json, block_id)
         end
       end
     end
+
+    def fetch_unconfirmed_deposits(latest_block, block_json = {})
```
Comment (ysv) [inline app/services/blockchain_service/bitcoin.rb]: ```Rails.logger.info { "Processing unconfirmed deposits." }```

**Review by ysv — CHANGES_REQUESTED** [review]

File: app/services/blockchain_service/bitcoin.rb (hunk 1 above)
Comment (dinesh-skyach) [reply app/services/blockchain_service/bitcoin.rb]: @ysv done

File: app/models/deposits/coin.rb [hunk 2]
```diff
@@ -30,7 +30,7 @@ def latest_block_number
 
     def confirmations
       return 0 if block_number.blank?
-      return latest_block_number - block_number if (latest_block_number - block_number) >= 0
```
Comment (ysv) [inline app/models/deposits/coin.rb]: this is wrong for eth leave as it was

File: app/models/withdraws/coin.rb [hunk 3]
```diff
@@ -38,7 +38,7 @@ def latest_block_number
 
     def confirmations
       return 0 if block_number.blank?
-      return latest_block_number - block_number if (latest_block_number - block_number) >= 0
```
Comment (ysv) [inline app/models/withdraws/coin.rb]: this is wrong for eth leave as it was we will solve this later

**Review by ysv — CHANGES_REQUESTED** [review]

**Comment by ysv** [comment]
@dinesh-skyach please rebase

File: app/models/deposits/coin.rb (hunk 2 above)
Comment (dinesh-skyach) [reply app/models/deposits/coin.rb]: @ysv why is it wrong for eth? We have to include the block in which the txn is included.

Comment (ysv) [reply app/models/deposits/coin.rb]: latest_block_number for eth returns number of current block ( block which is mining right now)
but for btc it returns block with is mining right now - 1

**Review by ysv — CHANGES_REQUESTED** [review]

**Review by ysv — APPROVED** [review]

**Review by mod — APPROVED** [review]

**Pull request merged**